src/components/Timeline.vue

```diff
@@ -1,185 +1,184 @@
 <template>
-  <div ref="visualization"></div>
+    <div ref="visualization"></div>
 </template>
 
 <script>
 import vis from 'vis';
 
 const events = [
-  'click',
-  'contextmenu',
-  'doubleClick',
-  'drop',
-  'mouseOver',
-  'mouseDown',
-  'mouseUp',
-  'mouseMove',
-  'groupDragged',
-  'changed',
-  'rangechange',
-  'rangechanged',
-  'select',
-  'itemover',
-  'itemout',
-  'timechange',
-  'timechanged',
+    'click',
```

Comment:
As a whole it looks good but is the indentation align with eslint airbnb-base rules ?